manager/src/main/java/com/redhat/service/bridge/manager/ProcessorService.java

```diff
@@ -23,4 +26,12 @@
     ListResult<Processor> getProcessors(String bridgeId, String customerId, int page, int size);
 
     void deleteProcessor(String bridgeId, String processorId, String customerId);
+
+    ProcessorDTO toDTO(Processor processor);
+
+    ProcessorResponse toResponse(Processor processor);
+
+    JsonNode definitionToJsonNode(ProcessorDefinition definition);
```

Comment:
Do `definitionToJsonNode` and `jsonNodetoDefinition` need to be declared on the interface? AFAICS you only use them internally to the implementation  of the `ProcessorServiceImpl`.

Reply:
They were used also in a test but I managed to make it work without the need to add these methods to the interface as well :+1: